rubocop-rspec.yml

```
@@ -12,3 +12,9 @@ RSpec/AnyInstance:
 
 RSpec/MessageSpies:
   Enabled: false
+
+RSpec/EmptyExampleGroup:
+  Enabled: true
+  Exclude:
+    - "spec/requests/**/*.rb"
```

```suggestion
    - "spec/requests/**/*.rb"
    + "spec/requests/**/*"
    + "test/requests/**/*"
```

This is how the default rubocop.yml treats this (avoiding the assumption of a `spec` folder)
https://raw.githubusercontent.com/rubocop/rubocop/master/config/default.yml

---

sounds reasonable
will update it

---

but in general, we don't have rspec everywhere

---

@Drowze ^

---

Actually nevermind. Just occurred to me that this is `rubocop-rspec` (i.e. an **rspec-specific** confiugration)

I think it's fine to make the assumption of a `spec` folder then. Approved :+1: